src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/OutputFormatterWriteContext.cs

```diff
@@ -21,29 +21,19 @@ public class OutputFormatterWriteContext : OutputFormatterCanWriteContext
         /// <param name="objectType">The <see cref="Type"/> of the object to write to the response.</param>
         /// <param name="object">The object to write to the response.</param>
         public OutputFormatterWriteContext(HttpContext httpContext, Func<Stream, Encoding, TextWriter> writerFactory, Type objectType, object @object)
+            : base(httpContext)
         {
-            if (httpContext == null)
-            {
-                throw new ArgumentNullException(nameof(httpContext));
-            }
-
             if (writerFactory == null)
             {
                 throw new ArgumentNullException(nameof(writerFactory));
             }
 
-            HttpContext = httpContext;
             WriterFactory = writerFactory;
             ObjectType = objectType;
             Object = @object;
         }
 
         /// <summary>
-        /// Gets or sets the <see cref="HttpContext"/> context associated with the current operation.
-        /// </summary>
-        public virtual HttpContext HttpContext { get; protected set; }
```

Comment:
I was worried that this might've been a binary breaking change, but according to http://stackoverflow.com/a/1522718/682105, it looks like it isn't.